.github/workflows/ci.yml

```diff
@@ -18,6 +18,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
```

Comment:
I'm assuming this is what's causing the CI to fail. No need for these changes afaik